### ISSUE TITLE
[BUG] GET /selector constantly return empty

### ISSUE THREAD

**Issue by zhc00** [apache/shenyu#3992]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Steps To Reproduce

1. create selector
```bash
curl -L -X POST 'http://localhost:9095/selector' \
-H 'X-Access-Token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyTmFtZSI6ImFkbWluIiwiZXhwIjoxNjY0MTM2NjAxfQ.00by321c2cZJojQRdMWv9PIRvX9sAifGNkr0muuJTEQ' \
-H 'Content-Type: application/json' \
-H 'Cookie: JSESSIONID=8A1196978527AEE40984DEED8D7E508A' \
--data-raw '{"id":null,"name":"httpbin","type":"1","matchMode":"0","enabled":true,"continued":true,"handle":"[{\"upstreamUrl\":\"httpbin:80\"}]","sort":1,"pluginId":"5","loged":true,"selectorConditions":[{"paramType":"uri","operator":"=","paramName":"/","paramValue":"/anything"}]}
'
```
2. get selectors

```bash
curl -L -X GET 'http://localhost:9095/selector?currentPage=1&pageSize=10' \
-H 'X-Access-Token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyTmFtZSI6ImFkbWluIiwiZXhwIjoxNjY0MTM2NjAxfQ.00by321c2cZJojQRdMWv9PIRvX9sAifGNkr0muuJTEQ' \
-H 'Cookie: JSESSIONID=E4F70AA17F5BFCD302727AA66EAAD90F'
```

### Environment

```markdown
ShenYu version(s): master with h2
```

**Comment by li-keguo** [comment]
hi～，You must provide the plugin Id, this interface is somewhat ambiguous

```bash
curl -L -X GET 'http://localhost:9095/selector?currentPage=1&pageSize=10&pluginId=5' \                                        
-H 'X-Access-Token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyTmFtZSI6ImFkbWluIiwiZXhwIjoxNjY0MTM2NjAxfQ.00by321c2cZJojQRdMWv9PIRvX9sAifGNkr0muuJTEQ' \

```

This is a legacy issue, this interface will be deprecated in the future, the new interface is as follows:
```http

POST http://localhost:9095/selector/list/search

POST http://localhost:9095/selector/list/search/adaptor

```

see: http-debug-selector-controller-api.http

**Comment by moremind** [comment]
I will fix it in #4065

**Comment by moremind** [comment]
the bug has been fixed
